### DISCUSSION TITLE
FLAC mountpoint settings

### DISCUSSION THREAD

**Discussion by Gozzeh** [AzuraCast/AzuraCast#5850] [Q&A]
I've been messing about with a flac endpoint for true lossless broadcasting of live audio.  However I've hit a bit of an issue.

I've noticed that the FLAC stream is using around 1.7Mbps of bandwidth which is higher than the 1.56Mbps  requitred for 48kHz 16-bit stereo.

According to the liquid soap settings its set to use a bitdepth of 24.  VLC however says its set to a bit depth of 32 (which makes a it more sense from the bandwidth numbers).  How can I set this to run at a bitdepth of 16-bit?   It would save a tone of bandwidth.

**Comment by ivellios1988** [comment]
You may need to create a raw configuration for your FLAC stream.

**Reply by Gozzeh** [reply]
I assume you mean directly editing config files?  Can you point me in the direction of which files to edit?

Am a bit new to azuracast ...

**Reply by ivellios1988** [reply]
Of course you can edit liquidsoap.liq configuration file directly but this creates an issue - namely, the file will be overwritten each time you restart Liquidsoap and all changes you made to it will be removed.

A better idea will be to do this:

1) Go to Broadcasting -> Mount Points -> create a mountpoint with FLAC ('enable AutoDJ' should be checked).

2) Then go to Broadcasting -> Edit Liquidsoap Configuration. It will show you the contents of liquidsoap.liq configuration file. Search for a line starting with:

`output.icecast(%ogg(%flac`

It will contain the definition of your FLAC mountpoint. Of course it should also contain the name of your mountpoint.

Now all you need to do is just copy the whole line, paste it to the textarea at the bottom and change bits_per_sample to 16. Save changes.

3) Now go back to Broadcasting -> Mount Points and edit your FLAC mountpoint - now you need to uncheck the 'enable AutoDJ' option.

4) Click 'Reload to Apply Changes'. Now your FLAC stream should play at 16 bits.

**Reply by ivellios1988** [reply]
I made an instruction video to make your radio life even easier :)

https://user-images.githubusercontent.com/13646017/201533521-23adfb46-a110-45b3-8609-7193c8958dec.mp4

I forgot to enable Liquidsoap and Icecast before I started recording the video so I had to start them at the end.

**Reply by ivellios1988** [reply]
Hey hey, it works ❤️

![Bez tytułu](https://user-images.githubusercontent.com/13646017/201533734-88eaac02-8d7a-412c-a407-83dd31e4455b.png)

**Comment by Gozzeh** [comment]
Thanks!   Much apprecited.  i now have it streaming at 16-bit.
a
